src/modules/dashboard/components/Whitelist/Whitelist.tsx

```diff
@@ -1,10 +1,22 @@
+import { isEmpty } from 'lodash';
```

Comment:
I think Alicja mentioned in a previous PR that it's better to import from `lodash/isEmpty`